clients/admin-ui/src/features/common/tablev2/FidesTable.tsx

```diff
@@ -0,0 +1,157 @@
+import {
+  Table,
+  TableContainer,
+  Tbody,
+  Td,
+  Th,
+  Thead,
+  Tr,
+} from "@fidesui/react";
+import {
+  flexRender,
+  RowData,
+  Table as TableInstance,
+} from "@tanstack/react-table";
+import React, { ReactNode } from "react";
+
+const getTableTHandTDStyles = (cellId: string) =>
+  cellId === "select"
+    ? { padding: "0px" }
+    : {
+        paddingLeft: "16px",
+        paddingRight: "8px",
+        paddingTop: "0px",
+        paddingBottom: "0px",
+      };
+
+/*
+  This was throwing a false positive for unused paramaters.
+  It's also how the library author reccomends typing meta.
+  https://tanstack.com/table/v8/docs/api/core/column-def#meta
+*/
+/* eslint-disable */
+declare module "@tanstack/table-core" {
+  interface ColumnMeta<TData extends RowData, TValue> {
+    width?: string;
+  }
+}
+/* eslint-enable */
+
+type Props<T> = {
+  tableInstance: TableInstance<T>;
+  rowActionBar?: ReactNode;
+  footer?: ReactNode;
+  onRowClick?: (row: T) => void;
+};
+
+export const FidesTableV2 = <T,>({
+  tableInstance,
+  rowActionBar,
+  footer,
+  onRowClick,
+}: Props<T>) => (
+  <TableContainer
+    overflowY="auto"
+    borderBottomWidth="1px"
+    borderBottomColor="gray.200"
+  >
+    <Table
+      variant="unstyled"
+      style={{
+        borderCollapse: "separate",
+        borderSpacing: 0,
+      }}
```

Comment:
Styling it this way allows for a sticky header that doesn't have its borders move. `borderCollapse: collapse` has a weird issue where the headers borders will scroll away while the row itself remains sticky.